pallets/erc20-app/src/mock.rs.orig

```diff
@@ -0,0 +1,140 @@
+// Mock runtime
```

Comment:
here and later- are these .orig files created intentionally?

Reply:
my bad - fixed already